src/BugsnagUnity/Native/Android/NativeApp.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using BugsnagUnity.Payload;
+using UnityEngine;
+
+namespace BugsnagUnity
+{
+    public class NativeApp : NativePayloadClassWrapper, IApp
```

Comment:
Will making this `public` expose it to users as part of our API?

Reply:
Changed to internal